parsers/powershell.c

```diff
@@ -602,6 +618,41 @@ static bool parseClass (tokenInfo *const token)
 	return readNext;
 }
 
+/* parse a enum
+ *
+ * 	enum EnumName {}
+ */
+static bool parseEnum (tokenInfo *const token)
+{
+	bool readNext = true;
+	tokenInfo *nameFree = NULL;
+
+	readToken (token);
+
+	if (token->type != TOKEN_IDENTIFIER)
+		return false;
+
+	nameFree = newToken ();
+	copyToken (nameFree, token, true);
+	readToken (token);
+
+	makeEnumTag (nameFree);
+
+	while (token->type != TOKEN_OPEN_CURLY && token->type != TOKEN_EOF)
+	{
+		readToken (token);
+	}
+
+	if (token->type == TOKEN_OPEN_CURLY)
+		enterScope (token, nameFree->string, K_CLASS);
```

Comment:
Oh, I see. You need the string in this step.
In that case, you need only a vstring.
See main/vstring.h.

See (*1).

Reply:
(*2)
```
647: enterScope (token, nameFree, K_CLASS);
```

Reply:
I wonder why K_CLASS. Don't you use K_ENUM here?

Reply:
Thank you for teaching me about vString.

We should use K_ENUM.

Please let me fix it.